sources/osgShader/ShaderProgramBuilder.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
```

Comment:
filename is not well named. ShaderProgramBuilder means it's a class. shaderProgramBuilder means it exposes functions / lib